lib/rspec/rails/view_rendering.rb

```diff
@@ -64,17 +64,19 @@ def self.nullify_template_rendering(templates)
         # @private
         class LogSubscriber < ::ActiveSupport::LogSubscriber
           def current_example_group
-            RSpec.current_example.example_group
+            # When running feature specs current_example can be nil if the subscriber runs in a different context
+            # than the specs themselves. E.g. in a capybara thread.
+            RSpec.current_example.try!(:example_group)
```

Comment:
Please don't use `try!` here, we try to avoid the usage of non standard Ruby, you could add a guard to return nil if theres no current example but really we should figure out why its nil.